src/libra_py/CP2K_methods.py

```diff
@@ -161,31 +161,33 @@ def cube_file_names_cp2k( static_or_dynamics, project_name, min_band, max_band,
 
 
 
-def read_cp2k_tddfpt_log_file( tddfpt_log_file_name, num_ci_states, tolerance ):
+def read_cp2k_tddfpt_log_file( params ):
     """
     This function reads log files generated from TD-DFPT calculations using CP2K and returns the TD-DFPT
     excitation energies, Slater determinent states in terms of the Kohn-Sham orbital indicies, 
     and the Slater determinent coefficients that comprise the multi-configurational electronic states
     
     Args:
-
-        tddfpt_log_file_name ( string ): name of the log file for this particular timestep
-
-        num_ci_states ( int ): how many ci states to consider
-
-        tolerance ( float ): the tolerance for accepting SDs are members of the CI wavefunctions
-
+      params ( dictionary ): parameters dictionary 
+          logfile_name ( string ): name of the log file for this particular timestep
+          num_ci_states ( int ): how many ci states to consider
```

Comment:
lets not call "ci_states", just call them "states", it is implied that when you do TD-DFT, those are the multiconfigurational states. It is not entirely correct to refer to the TD-DFT states as CI states.
So, please change the variable name to "nstates" or "number_of_states" in the input, and potentially in the implementation